src/mono/wasm/debugger/BrowserDebugHost/BrowserDebugHost.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Why not `$(NetCoreAppToolCurrent)`?

Reply:
I didnt use ```$(NetCoreAppToolCurrent)``` bcoz these projects are not tools. The restriction  here to not able to use ```$(NetCoreAppCurrent)``` is that we dont have a local aspnetcore targetingpack with net6.0 paths.

Reply:
Makes sense, then I would recommend to define an additional property like `AspNetCoreCurrent`.